-- Fix CMake 4.x compatibility by raising the minimum required CMake version.
-- CMake 4.0 dropped support for cmake_minimum_required(VERSION < 3.5).
--- cargo-crates/shaderc-sys-0.8.3/build/CMakeLists.txt.orig	2006-07-23 18:21:28.000000000 -0700
+++ cargo-crates/shaderc-sys-0.8.3/build/CMakeLists.txt	2026-08-27 15:50:07.194669000 -0700
@@ -3,7 +3,7 @@
 # of using the directory hierarchy required by shaderc, which Git submodule
 # does not like.
 
-cmake_minimum_required(VERSION 2.8.12)
+cmake_minimum_required(VERSION 3.5)
 
 project(native-shaderc)
 
